mkosi/config.py

```diff
@@ -499,7 +499,18 @@ def native(cls) -> "Architecture":
         return cls.from_uname(platform.machine())
 
 
-def parse_boolean(s: str) -> bool:
+class ArtifactOutput(enum.IntFlag):
+    uki = 1
+    kernel = 2
+    initrd = 4
+    partitions = 8
+
+    # Set a high bit to prevent these from showing up when only the previous values are set
+    no = 0b10000000000000000000000000000000 | uki | kernel | initrd
+    yes = 0b10000000000000000000000000000000 | uki | kernel | initrd | partitions
```

Comment:
I wonder if this is just too much C for python and we're better off just having a set of enum values instead. That makes the JSON serialization easier as well.